### PR TITLE
Release 1 10 1

### PR DESCRIPTION
**Pull request by OfirMos**

### CODE REVIEW

File: src/libmr_commands.c [hunk 1]
```diff
@@ -15,10 +15,22 @@
 static inline bool check_and_reply_on_error(ExecutionCtx *eCtx, RedisModuleCtx *rctx) {
     size_t len = MR_ExecutionCtxGetErrorsLen(eCtx);
     if (unlikely(len > 0)) {
-        RedisModule_ReplyWithError(rctx, "multi shard cmd failed");
         RedisModule_Log(rctx, "warning", "got libmr error:");
+        bool max_idle_reached = false;
         for (size_t i = 0; i < len; ++i) {
             RedisModule_Log(rctx, "warning", "%s", MR_ExecutionCtxGetError(eCtx, i));
+            if (!strcmp("execution max idle reached", MR_ExecutionCtxGetError(eCtx, i))) {
+                max_idle_reached = true;
```
Comment (oshadmi) [inline src/libmr_commands.c:23]: can add a break?

**Review by oshadmi — COMMENTED** [review]
Few comments

File: src/libmr_commands.c (hunk 1 above)
Comment (OfirMos) [reply src/libmr_commands.c:23]: @oshadmi I still would like to log all errors from libmr

File: src/module.c [hunk 2]
```diff
@@ -137,34 +137,19 @@ int TSDB_info(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
...
             }
 
-            if (reply_map) {
-                RedisModule_ReplyWithMap(ctx, 1);
-            }
             RedisModule_ReplyWithSimpleString(ctx, "startTimestamp");
             RedisModule_ReplyWithLongLong(
                 ctx, numOfSamples == 0 ? -1 : series->funcs->GetFirstTimestamp(chunk));
-            if (reply_map) {
-                RedisModule_ReplyWithMap(ctx, 1);
```
[the remark this answers is not in the document]
Comment (oshadmi) [reply src/module.c:150]: After further discussion:
5 Maps with len 1 were replaced by 1 Map with len 5.
And this is an expected fix in this patch release.

**Review by oshadmi — APPROVED** [review]

**Pull request merged**